web/src/components/LandingPage/HeroSection.tsx

```diff
@@ -1,6 +1,7 @@
+import React from "react";
 import { Highlight } from "../ui/hero-highlight";
 
-const HeroSection = () => {
+const HeroSectionComponent = () => {
```

Comment:
This is mostly a matter of personal preference, but I think it is preferable not to use suffixes like "...Component", "...Class", "...Variable" when naming things.